templates/partner/_slider.html.twig

```diff
@@ -1,35 +1,26 @@
-<div class="container">
-    <div class="row  d-flex justify-content-center">
-        <h2 class="title_partners ">Nos partenaires</h2>
-    </div>
-
-    <div class="row justify-content-center">
-        <div id="carouselExampleFade" class="carousel slide carousel-fade" data-ride="carousel">
-            <div class="carousel-inner">
+{% if partners|length > 0 %}
+    <div class="container-fluid bg-dark">
+        <div class='container-slider container'>
+            <div class='single-item'>
                 {% for partner in partners %}
-                <div class="carousel-item
-                {% if loop.first %} active{% endif %}
-                ">
-
-                    <a href="{{ partner.url }}">
-                        <img class="d-block slider-patners" src="{{ partner.picture }}" alt="logo de {{ partner.name }}">
-                        <div class="carousel-caption d-none d-md-block">
-                            <p class="name-patner text-dark">{{ partner.name }}</p>
-                        </div>
-                    </a>
-
-                </div>
+                    <div class="">
+                        <a href="{{ partner.url }}" class="text-center mx-auto">
+                            <img class="carousel-icon align-content-center"
+                                 src="{{ asset("images/partners/") }}{{ partner.picture }}" alt="">
+                        </a>
+                    </div>
                 {% endfor %}
+                {% if partners|length < 5 %}
```

Comment:
J'ai surement mal compris ? Je vois deux fois la meme boucle à la suite {% for partner in partners %}. La seconde étant conditionnée au fait qu'il y ait moins de 5 elements.